Lib/fontTools/merge.py

```diff
@@ -424,9 +424,15 @@ def _glyphsAreSame(glyphSet1, glyphSet2, glyph1, glyph2):
 	g2 = glyphSet2[glyph2]
 	g1.draw(pen1)
 	g2.draw(pen2)
-	return (pen1.value == pen2.value and
-		g1.width == g2.width and
-		(not hasattr(g1, 'height') or g1.height == g2.height))
+	if pen1.value != pen2.value:
+		return False
+	tolerance = .05 if pen1.value else .20
+	if (g1.width - g2.width) > g1.width * tolerance:
```

Comment:
Doesn't this - and height below - give different results for the same two widths depending on whether g1 or g2 has larger/smaller value? - e.g. g1.w=100 g2.w=106 vs g1.w=106 g2.w=100.

I think I'd have expected something along the lines of `abs(g1.width - g2.width) > min(g1.width, g2.width) * tolerance` ?

Reply:
> 
Doesn't this - and height below - give different results for the same two widths depending on whether g1 or g2 has larger/smaller value? - e.g. g1.w=100 g2.w=106 vs g1.w=106 g2.w=100.

It does give some preference to the first font... which makes sense since the value for that one will be used...

Reply:
I also found that a bit weird. this is basically saying the glyphs are different if the second has slightly wider/taller advance, but same if slightly narrower. I don't immediatey see the logic.

Reply:
Right. Meant to have `abs` in there, just not `min`.